src/js/components/widget.jsx

```diff
@@ -49,6 +53,11 @@ export class WidgetComponent extends Component {
 
     onClick = () => {
         resetUnreadCount();
+        this.setState({showAnimation: true});
```

Comment:
You could even not have an action to do it and just have the reducer turn on the flag when the action is dispatched. You would end up with only a `disableAnimation` action.